Deceive/Utils.cs

```diff
@@ -127,9 +127,9 @@ public static string GetLCUPath()
 
         /**
          * Asynchronously checks if the current version of Deceive is the latest version. If not, and the user has
-         * not dismissed the message before, an alert is shwon.
+         * not dismissed the message before, an alert is shown.
          */
-        public static async Task CheckForUpdates()
+        public static async void CheckForUpdates()
```

Comment:
If we don't want to await and don't need anything this method returns just 'run it and forget it'.